roles/core/ssh_master/templates/config.j2

```diff
@@ -1,9 +1,13 @@
+#jinja2: lstrip_blocks: "True"
 {% import "macros/network.j2" as macro_network with context %}
 {# Macro to avoid redundancy #}
 {% macro write_host(host,macro_node_main_network) %}
 Host {{host}}
+    {% if ssh_master_disable_hosts_check is defined and
+          ssh_master_disable_hosts_check == true %}
```

Comment:
setting disable to true is quite disturbing, can you rename this variable to something more accurate like :
ssh_master_hostkey_check = true|false

Or 

enable_hostkey_check

Also, it should be present in the inventory (with hostkey check set to no to ensure backward compatibility)

Reply:
I want to make it clear to the sysadmin that they are disabling hostkey check, which is a security concern. `ssh_master_disable_hostkey_check` might be better wording though.

> Also, it should be present in the inventory (with hostkey check set to no to ensure backward compatibility)

I don't want it to be backward compatible, hence the `breaking change` label. I believe it's better to provide more security by default and let user remove security on purpose.

Reply:
I understand both.

Please give me few hours, I might have an idea regarding this and the future of the security side of the stack. :-)

Reply:
A new file, group_vars/all/security.yml will be added.

```
---
security:
  ssh:
    hostkey_check: true # enable or disable hostkey check
```

Since many security features will come with next releases, having a dedicated file is better for sys admin to understand the risks.

Reply:
So no `disable` in the variable name to explicit the disabling of a security mechanism?

Reply:
I think the fact that these parameters are set in this file, called security, will warn users manipulating these parameters has to be done with care.
We should even add an ASCII Warning art, to inform this file is security key.

Reply:
nagios values are return codes, not boolean. Any application in Unix should return 0 when it completes successfully.
Anyway I updated the PR to use security.ssh.hostkey_checking.